hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -188,6 +188,9 @@ public final class OzoneConfigKeys {
       "ozone.client.ec.grpc.write.timeout";
   public static final String OZONE_CLIENT_EC_GRPC_WRITE_TIMEOUT_DEFAULT = "30s";
 
+  public static final String OZONE_EC_GRPC_ZERO_COPY_ENABLED = "ozone.ec.grpc.zerocopy.enabled";
+  public static final boolean OZONE_EC_GRPC_ZERO_COPY_ENABLED_DEFAULT = false;
```

Comment:
Why do we need a feature flag for this? This requires us maintaining and testing two different code paths.

Reply:
Good point. I'm also not sure about this and open for suggestions. Guess it's worth giving clients a chance to disable this optimization so that they don't need to rollback an entire major ozone version (say ozone 1.4 or 1.5) if running into problems with zero-copy. 
I think we should have it enabled by default to have all the existing integration tests go through it.